torch/_inductor/kernel/fused_int_mm_mul.py

```diff
@@ -0,0 +1,97 @@
+import logging
+from typing import List
+
+from ..select_algorithm import autotune_select_algorithm, ChoiceCaller, TritonTemplate
+from .mm_common import int8_mm_configs, mm_args, mm_grid, mm_options
+
+log = logging.getLogger(__name__)
+
+fused_int_mm_mul_template = TritonTemplate(
```

Comment:
You shouldn't need a new template for adding an epilogue to an existing template.  

See how addmm uses `epilogue_fn`:
https://github.com/pytorch/pytorch/blob/ad4472833ce5f40ca595492bd48f6735c69d6a3f/torch/_inductor/kernel/mm.py#L240

Reply:
thanks, I had been looking for an example of how to do that